docs/en/api/POSIX-API.md

```diff
@@ -260,46 +262,61 @@ alluxio.job.worker.threadpool.size=64
 
 ### Cache Tuning
 
-When Alluxio system (master and worker) can provide metadata/data cache to speed up the metadata/data access of Alluxio under storage files/directories,
-Alluxio FUSE can provide another layer of local metadata/data cache on the application nodes to further speed up the metadata/data access.
+When an application runs an operation against the local FUSE mount point.
+The request will be processed by FUSE kernel, Fuse process, and Alluxio system sequentially.
+If at any level, cache is enabled and cache hit, cached metadata/data will be returned to the application without going through the whole process to improve the overall read performance.
+
+While Alluxio system (master and worker) provides remote distributed metadata/data cache to speed up the metadata/data access of Alluxio under storage files/directories,
+Alluxio FUSE provides another layer of local metadata/data cache on the application nodes to further speed up the metadata/data access.
 
-Alluxio FUSE can provide two kinds of metadata/data cache, the kernel cache and the userspace cache. Kernel cache is executed by Linux kernel
-and metadata/data are stored in operating system kernel cache. Userspace cache is controlled and managed by Alluxio FUSE process
-and metadata/data are stored in user configured location (process memory for metadata, ramdisk/disk for data).
+Alluxio FUSE can provide two kinds of metadata/data cache, the kernel cache and the userspace cache.
+Kernel cache is executed by Linux kernel with metadata/data stored in operating system kernel cache.
+Userspace cache is controlled and managed by Alluxio FUSE process with metadata/data stored in user configured location (process memory for metadata, ramdisk/disk for data).
+Although both kinds of cache can be enabled at the same time, it’s recommended to choose only one of them to avoid double memory consumption.
```

Comment:
if you take the below recommendation to swap the two blocks, then this sentence can combine into the other block. can be something like... 

Since FUSE kernel cache and userspace cache both provide caching capability, although they can be enabled at the same time, it is recommended to choose only one of them  to avoid double memory consumption. Here is a guideline on how to choose between the two cache types based on your environment and needs.